### PR TITLE
add navbar and profile

### PR DESCRIPTION
**Pull request by vermaankit484**

### CODE REVIEW

**Comment by vermaankit484** [comment]
[Link to UI](https://drive.google.com/file/d/1zkeBTdAcT49FTaxJI5gpMIG2UdcayAwq/view?usp=sharing)

**Comment by rishikagupta2468** [comment]
Just a suggestion, in script.js
Instead of adding a separate.click function for every navbar button, you can use navbar li and bind click event with that
It will reduce code duplication, for reference, you can have a look on my script.js code for navbar

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -3,16 +3,56 @@
...
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body onload="typeName()">
```
Comment (cpaliwal) [inline portfolio/src/main/webapp/index.html:14]: prefer document.addEventListener('DOMContentLoaded', fn) or window.addEventListener('load', fn) over this

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,17 +12,40 @@
...
+});
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+let idx=0;
+let name='Ankit Verma';
+typeName = () => {
+    if (idx < name.length) {
+        document.getElementById('name').innerHTML += name.charAt(idx);
```
Comment (cpaliwal) [inline portfolio/src/main/webapp/script.js:40]: for consistency you can use jquery selector  but this is fine too

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,40 @@
...
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+$(document).ready(function() {
+    $("#bodynav li").click(function() {
+        $(this).addClass("active");
+        $(this).siblings().removeClass("active");
+        $("#headnav li").removeClass("active");
+    })
+    $("#headnav li").click(function() {
```
Comment (cpaliwal) [inline portfolio/src/main/webapp/script.js:22]: why is behaviour different for this and above. Can this be combined by using some class?

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -3,16 +3,56 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="utf-8">  
+    <meta name="viewport" content="width=device-width, initial-scale=1">  
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">  
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>  
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
```
Comment (cpaliwal) [inline portfolio/src/main/webapp/index.html:10]: generally recommended to put scripts at bottom

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -3,16 +3,56 @@
...
+                <div class="row">
+                    <div class="col-sm-5">
+                        <img src="images/photo_2020-07.jpg" height="80%" width="70%">
+                    </div>
+                    <div class="col-sm-7">
+                        <div>I am pursuing Bachelors of Engineering CS from Chandigarh University, batch 2021. 
+                        Inclined towards Problem Solving, so can be mostly seen on platforms like Codeforces, Codechef, Atcoder with handle <span style="color: darkmagenta">verma_ankit484</span>.<br>
+                        Besides this, working on projects, I find Web Development as part of my interest. I always seek to find best out of an opportunity. Participating in Competitions, Hackathons experiencing the adrenaline rush is what motivates me the most.
+                        </div>
+                        <div style="padding-top: 40px">
```
Comment (cpaliwal) [inline portfolio/src/main/webapp/index.html:49]: avoid inline style

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (vermaankit484) [reply portfolio/src/main/webapp/script.js:22]: the list elements of bodynav class are supposed to be collapsed, while About me should remain there, that's why they are in different div

**Pull request merged**